test multi/017eda4c::fail1[0]: FAILED
test multi/017eda4c::fail2[0]: FAILED
test multi/017eda4c::fail3[0]: FAILED

failures:

---- multi/017eda4c::fail1[0] counterexamples ----
[Crux] Found counterexample for verification goal
[Crux]   test/symb_eval/crux/multi.rs:8:22: 8:27: error: in multi/017eda4c::fail1[0]
[Crux]   attempt to compute `move _5 + const 1_u8`, which would overflow
[Crux] Found counterexample for verification goal
[Crux]   ./lib/crucible/lib.rs:38:41: 38:58 !test/symb_eval/crux/multi.rs:8:5: 8:32: error: in multi/017eda4c::fail1[0]
[Crux]   MIR assertion at test/symb_eval/crux/multi.rs:8:5:
[Crux]   	x + 1 > x

---- multi/017eda4c::fail2[0] counterexamples ----
[Crux] Found counterexample for verification goal
[Crux]   internal: error: in multi/017eda4c::fail2[0]
[Crux]   panicking::panic_fmt, called from multi/017eda4c::fail2[0]

---- multi/017eda4c::fail3[0] counterexamples ----
[Crux] Found counterexample for verification goal
[Crux]   ./lib/crucible/lib.rs:38:41: 38:58 !test/symb_eval/crux/multi.rs:20:5: 20:29: error: in multi/017eda4c::assert_zero[0]
[Crux]   MIR assertion at test/symb_eval/crux/multi.rs:20:5:
[Crux]   	x == 0

[Crux] Overall status: Invalid.
